docs/reference/types.md

```diff
@@ -0,0 +1,81 @@
+(types)=
+# Type format
+
+SQLair supports structs maps and slices in input expressions and structs and
```

Comment:
structs,* maps,* and

Reply:
I'd also structure in two bullet points:

SQLair supports the following types:

- in input expressions:
    - structs 
    - maps
    - slices
 - in output expressions:
    - structs
    - maps

The nested lists are optional (as in, you can give them in-line in the host list item).